site-src/v1alpha2/guides/tcp.md

```diff
@@ -14,17 +9,50 @@ distribute the traffic with the following rules:
 - All TCP streams on port 8090 of the Gateway are forwarded to port 6000 of
   `my-bar-service` Kubernetes Service.
 
-Please note the following:
+In this example two `TCP` listeners will be applied to the [Gateway][gateway]
```

Comment:
The gateway link is undefined.

Reply:
fixed in c20ba93a26d200a1e0292d6b3800c3f46e264890 :+1: